src/phoenix/experimental/evals/models/openai.py

```diff
@@ -72,13 +72,13 @@ def _generate(self, prompt: str, instruction: Optional[str]) -> str:
         invoke_params = self.invocation_params
         messages = [{"role": "user", "content": prompt}]
         if instruction:
-            messages.append({"role": "system", "content": instruction})
+            messages.insert(0, {"role": "system", "content": instruction})
         response = self._generate_with_retry(
             messages=messages,
             **invoke_params,
         )
         # TODO: This is a bit rudimentary, should improve
-        resp_text = str(response["choices"][0]["message"]["content"]).strip()
+        resp_text = str(response["choices"][0]["message"]["content"])
```

Comment:
was the strip needed?

Reply:
It was stripping the raw text output by the LLM. Seems to me like that logic belongs in the parser, not in the LLM model itself.